### PR TITLE
Refactoring and cleanup

### PR DESCRIPTION
**Pull request by draychev**
While reviewing all the code in this repository, I took the liberty to make a few renames, reorganizations, and refactors.

- Where imports could take one line - I changed it to be on a single line.
- Moved `pkg/kuberneteshelper` into `pkg/kubernetes/pod`
- Expanded on some of the `(add example)` TODOs
- Deleted unused `EnvVars()` function
- Moved the `Runnable` interface from `pkg/common` to `pkg/runner`
- Changed type from `string` to `common.MeshNamespace` anywhere we need OSM Controller's namespace
- Shortened logger names: `log = logger.New("osm-health/connectivity")` became `var log = logger.New("connectivity")`
- Deleted `cmd/collect.go` since it is not used anywhere at the moment

Signed-off-by: Delyan Raychev <delyan.raychev@microsoft.com>

### CODE REVIEW

File: pkg/printer/table.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+				log.Error().Err(err)
+				return
+			}
+		}
+	}
+
+	var statusIcon = "✅"
+	if errorsCount > 0 {
+		statusIcon = "❌"
+	}
```
Comment (johnsonshi) [inline pkg/printer/table.go:45]: I thought we were removing emojis?

File: cmd/connectivity_pod_to_pod.go [hunk 2]
```diff
@@ -30,19 +29,19 @@ func newConnectivityPodToPodCmd() *cobra.Command {
 				return errors.Errorf("provide both SOURCE_POD and DESTINATION_POD")
 			}
 
-			fromPod, err := kuberneteshelper.PodFromString(args[0])
+			sourcePod, err := pod.FromString(args[0])
```
Comment (shalier) [inline cmd/connectivity_pod_to_pod.go:32]: For consistency w/ the rest of the repo, can we change `sourcePod` to `srcPod` ?

File: cmd/connectivity_pod_to_pod.go [hunk 3]
```diff
@@ -30,19 +29,19 @@ func newConnectivityPodToPodCmd() *cobra.Command {
...
 			if err != nil {
 				return errors.New("invalid SOURCE_POD")
 			}
 
-			toPod, err := kuberneteshelper.PodFromString(args[1])
+			destinationPod, err := pod.FromString(args[1])
```
Comment (shalier) [inline cmd/connectivity_pod_to_pod.go:37]: For consistency w/ the rest of the repo, can we change `destinationPod` to `dstPod` ?

File: cmd/connectivity_pod_to_pod.go [hunk 4]
```diff
@@ -30,19 +29,19 @@ func newConnectivityPodToPodCmd() *cobra.Command {
...
 			if err != nil {
 				return errors.New("invalid DESTINATION_POD")
 			}
 
 			osmControlPlaneNamespace := settings.Namespace()
 
-			connectivity.PodToPod(fromPod, toPod, osmControlPlaneNamespace)
+			connectivity.PodToPod(sourcePod, destinationPod, osmControlPlaneNamespace)
```
Comment (shalier) [inline cmd/connectivity_pod_to_pod.go:44]: sourcePod -> srcPod
destinationPod -> dstPod

File: cmd/connectivity_pod_to_url.go [hunk 5]
```diff
@@ -32,7 +31,7 @@ func newConnectivityPodToURLCmd() *cobra.Command {
 				return errors.Errorf("provide both SOURCE_POD and DESTINATION_URL")
 			}
 
-			fromPod, err := kuberneteshelper.PodFromString(args[0])
+			fromPod, err := pod.FromString(args[0])
```
Comment (shalier) [inline cmd/connectivity_pod_to_url.go:34]: fromPod -> srcPod

Comment (shalier) [reply cmd/connectivity_pod_to_url.go:34]: Also, for consistency, in L47 also changing `fromPod` to `srcPod` 
https://github.com/openservicemesh/osm-health/blob/main/cmd/connectivity_pod_to_url.go#L47

File: pkg/connectivity/pod-to-url.go [hunk 6]
```diff
@@ -7,15 +7,17 @@ import (
...
-	"github.com/openservicemesh/osm-health/pkg/kuberneteshelper"
+	"github.com/openservicemesh/osm-health/pkg/kubernetes/pod"
 	"github.com/openservicemesh/osm-health/pkg/osm"
+	"github.com/openservicemesh/osm-health/pkg/printer"
+	"github.com/openservicemesh/osm-health/pkg/runner"
 )
 
 // PodToURL tests the connectivity between a source pod and destination url.
-func PodToURL(fromPod *corev1.Pod, destinationURL *url.URL, osmControlPlaneNamespace string) {
+func PodToURL(fromPod *corev1.Pod, destinationURL *url.URL, osmControlPlaneNamespace common.MeshNamespace) {
```
Comment (shalier) [inline pkg/connectivity/pod-to-url.go:17]: fromPod -> srcPod

File: pkg/printer/table.go (hunk 1 above)
Comment (draychev) [reply pkg/printer/table.go:45]: I just asked my editor to move the code from one package to another - did not rewrite this.
So the content is the same.

File: cmd/connectivity_pod_to_pod.go [hunk 7]
```diff
@@ -5,17 +5,16 @@ import (
...
 const connectivityPodToPodDesc = `
-Checks connectivity between Kubernetes pods
-	(add more descriptive description)
+Checks connectivity between two Kubernetes pods
 `
 
 const connectivityPodToPodExample = `
 Example:
-	(add example)
+	$ osm-health connectivity pod-to-pod namespace-a/pod-x namespace-b/pod-b
```
Comment (SanyaKochhar) [inline cmd/connectivity_pod_to_pod.go:17]:
```suggestion
	$ osm-health connectivity pod-to-pod namespace-a/pod-x namespace-b/pod-y
```

File: pkg/printer/table.go (hunk 1 above)
Comment (draychev) [reply pkg/printer/table.go]: Removed!

**Comment by draychev** [comment]
ptal

File: cmd/connectivity_pod_to_pod.go (hunk 7 above)
Comment (draychev) [reply cmd/connectivity_pod_to_pod.go]: Thanks for the hint!  Changed this to namespace-a/pod-a  and namespace-b/pod-b for clarity

**Review by shalier — APPROVED** [review]

**Pull request merged**